.github/workflows/shared-build-and-test-stronghold-nodejs.yml

```diff
@@ -144,3 +145,14 @@ jobs:
             if-no-files-found: error
             retention-days: 1
 
+        - name: Upload dist artifact
+          if: ${{inputs.upload-binaries-as-artifacts && matrix.settings.generate-dist}}
+          uses: actions/upload-artifact@v2
+          with:
+            name: ${{ inputs.output-artifact-name }}-dist
+            path: |
+              ${{ env.WORKING_DIRECTORY }}/dist/
+              !${{ env.WORKING_DIRECTORY }}/dist/${{ env.APP_NAME }}.*.node
+            if-no-files-found: error
+            retention-days: 1
```

Comment:
Maybe comment that this just uploads the js and ts files and only needs to be done on one platform.